.github/ISSUE_TEMPLATE/10-bug-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "🐛 Bug Issue"
+about: Use this template for reporting a bug
+title: ''
+labels: bug
+assignees: ''
```

Comment:
Are assignees github users or could we use mailing addresses ? It would be good to send notifications automatically for triage groups , although maybe we can use just tag following for that...

Reply:
I could not found an option to do that.